fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -1410,7 +1432,7 @@ ACTOR Future<Void> tLogPopCore(Reference<TLogGroupData> self,
 			    .detail("StorageTeam", storageTeamData->storageTeamId)
 			    .detail("UpTo", upTo)
 			    .detail("PoppedVersionLag", PoppedVersionLag)
-			    // TODO: add minPopStorageTeam after having popDiskQueue()
+			    .detail("MinPoppedStorageTeamVersion", logData->minPoppedStorageTeamVersion)
 			    .detail("QueuePoppedVersion", logData->queuePoppedVersion)
 			    .detail("UnpoppedRecovered", storageTeamData->unpoppedRecovered ? "True" : "False")
```

Comment:
Nit: `.detail("UnpoppedRecovered", storageTeamData->unpoppedRecovered)` should also work, and will print 0 or 1 as values.